crates/services/p2p/src/gossipsub/config.rs

```diff
@@ -25,9 +25,57 @@ use sha2::{
 };
 use std::time::Duration;
 
-use super::topics::GossipTopic;
+use super::topics::{
+    GossipTopic,
+    CON_VOTE_GOSSIP_TOPIC,
+    NEW_BLOCK_GOSSIP_TOPIC,
+    NEW_TX_GOSSIP_TOPIC,
+};
+
+// The number of slots in each epoch.
+const SLOTS_PER_EPOCH: u64 = 32;
+
+// The duration of each slot in seconds. This is the amount of time allotted for each opportunity to create a block.
+const SLOT: Duration = Duration::from_secs(6);
```

Comment:
What is this "mesh" and why previously used 1 millisecond and now we want to use 6 seconds?

Reply:
mesh is the gossipsub mesh, time in mesh is the time since the peer has been part of the gossipsub mesh, previously we used the default value, but now we want a more realistic value for our usecase, which would be at least the length between two blocks